Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -289,7 +289,12 @@ define([
      * @param {FrameState} frameState The frame state.
      */
     GlobeSurfaceTileProvider.prototype.initialize = function(frameState) {
-        this._imageryLayers._update();
+        var imageryLayers = this._imageryLayers;
+
+        // update collection: imagery indices, base layers, raise layer show/hide event
+        imageryLayers._update();
+        // update each layer for texture reprojection.
+        imageryLayers.update(frameState);
```

Comment:
It's kind of dodgy to have two update methods on `ImageryLayer`.  How about calling `update` `queueReprojectionCommands` cause that's what it actually does, right?